Documentation/services/pcn-packetcapture/Packetcapture.rst

```diff
@@ -0,0 +1,136 @@
+Packetcapture service
+=====================
+
+This transparent service allows you to capture the packets of the interface to which it is attached, apply filters and obtain capture in *.pcap* format. In particular, the service allows to get the capture in the local filesystem or it can communicate with a remote client which will receive the dump.
+
+An example of a client that uses the REST api of the packetcapture service is present in '*Packetcapture_Client*' direcotry.
+
+Features
+--------
+- Transparent Packetcapture service
+- IPv4 support
+- Support: source prefix, destination prefix, source port, destination port and layer 4 protocol filters.
+- Support snaplen
+- Support localmode or network mode operation
+
+Limitations
+-----------
+- Unsupported IPv6
+
+How to use
+----------
+The packetcapture service is a transparent service, it can be attached to a cube port.
+
+Create the service
+^^^^^^^^^^^^^^^^^^^^^
+
+
+::
+
+    #create the packetcapture service
+    polycubectl packetcapture add sniffer capture=bidirectional
+
+the packet capture service can capture packets in three differents ways:
+
+- ingress only: **capture=ingress**
+- egress only: **capture=egress**
+- bidirectional: **capture=bidirectional**
+- not working mode: **capture=off**
```

Comment:
a little confusing: this is "networking mode" or "not working mode"? I see there is networking mode in later section.

Reply:
Hi @goldenrye ,

this option indicates the direction of the packages that the service must capture.
The direction of the captured packets is independent of the operation in "nework mode" or non "network mode" of the service.


For example, we would like to be able to capture only incoming traffic to the service using the option:
_polycubectl sniffer set capture=ingress_

Reply:
so the line 38 should be "networking mode" instead of "not working mode", right?

Reply:
Jianwen, actually not, line 38 is just for turning capture off. I modified the code now, please check if now it looks better.

Reply:
@frisso Thank you for your commit